### PR TITLE
delete specialclick

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
delete specialclick
## Why It's Good For The Game
This was dumb.
We had some things coded onto CTRL click
And then we had other things coded onto specialclick (Which is defaulted to CTRL)
???
I'm deleting specialclick in favor of keeping our click code.
## Changelog
:cl:
del: Deletes "specialclick" keybind
/:cl:

### CODE REVIEW

File: strings/tips/marine.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ Blue weeds and purple sticky resin slow you down. Get rid of them!
 During Self Destruct, green control rods are armed control rods. Each rods take 3 minutes to rise, forcing marines to hold Self Destruct for a total of 20 minutes. Collect all six!
 A fit marine can carry many weapons: in their armor, belt, even hanging from their back.
 Requisitions has many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's in store and you may be pleasantly surprised. DISCLAIMER: Extra gears' availability is dependent on supply.
-As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL+click) on them or drag-clicking yourself onto them.
+As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by CTRL+click on them or drag-clicking yourself onto them.
```
Comment (coldironwarrior) [inline strings/tips/marine.txt:46]: `by using CTRL+click on them` or `by CTRL+clicking`

**Comment by TellyLsc** [comment]
Why? specialclick can be rebinded while the click code stuff can't, right? What if I want to climb windows by holding alt+click on window which is only possible because it's rebindable?

File: strings/tips/marine.txt (hunk 1 above)
Comment (ivanmixo) [inline strings/tips/marine.txt:46]:
```suggestion
As a marine, you can climb over waist-high obstacles like sandbags, window frames or tables by CTRL+clicking them or drag-clicking yourself onto them.
```

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Review by ivanmixo — APPROVED** [review]

**Comment by Al-1ce** [comment]
Really not a fan of this.

**Pull request merged**